requirements.txt

```diff
@@ -0,0 +1,27 @@
+# ray core deps
```

Comment:
do we want both a pipfile and a requirements.txt ?

Reply:
no , we dont need both.
i have removed req.txt